crypto/ocsp/ocsp_vfy.c

```diff
@@ -118,10 +118,6 @@ int OCSP_basic_verify(OCSP_BASICRESP *bs, STACK_OF(X509) *certs,
                 goto end;
             if (!X509_add_certs(untrusted, certs, X509_ADD_FLAG_DEFAULT))
                 goto end;
-        } else if (certs != NULL) {
-            untrusted = certs;
-        } else {
-            untrusted = bs->certs;
```

Comment:
How is this change related?

Reply:
This was poor man's concatenation of two cert lists in case either was empty.

Reply:
So it does not matter that untrusted will be NULL in case flags has OCSP_NOCHAIN  set?

Reply:
The `untrusted` list gets just passed to `X509_STORE_CTX_init()` as last parameter, which allows NULL.